app/helpers/application_helper/button/instance_evacuate.rb

```diff
@@ -1,10 +1,6 @@
 class ApplicationHelper::Button::InstanceEvacuate < ApplicationHelper::Button::Basic
-  def calculate_properties
-    super
-    self[:title] = @record.unsupported_reason(:evacuate) if disabled?
-  end
-
   def disabled?
-    !@record.supports_evacuate?
+    @error_message = @record.unsupported_reason(:evacuate) unless @record.supports_evacuate?
```

Comment:
Maybe instead of this button, [GenericFeatureButtonWithDisable](https://github.com/ManageIQ/manageiq/blob/master/app/helpers/application_helper/button/generic_feature_button_with_disable.rb) can be used